source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -798,16 +832,35 @@ public List<PimsAcquisitionFile> GetAcquisitionSubFiles(long acquisitionFileId,
         /// The digit base number is unique to the file. Do not pad the number with zeros.
         /// </item>
         /// <item>
-        /// Suffix - (ZZ above) The suffix numbers for an Acquisition file defaults to 01.
+        /// Suffix - (ZZ above) The suffix numbers for an Acquisition file defaults to 01 for "Main Files".
         /// </item>
         /// </list>
         /// </remarks>
         /// <returns>The formatted Acquisition File Number.</returns>
-        private static string GenerateAcquisitionNumber(short prefix, long fileNumber, short suffix = 1)
+        private static string GenerateAcquisitionNumber(short prefix, long fileNumber, int suffix = 1)
         {
             return $"{prefix:00}-{fileNumber}-{suffix:00}";
         }
 
+        /// <summary>
+        /// Returns the suffix portion of the supplied Acquisition File Number.
+        /// </summary>
+        /// <param name="acquisitionFileNumber">The Acquisition File Number.</param>
+        /// <returns>The file number suffix (e.g. "1", "2", "3", etc) if it is found, or -1 if it is not.</returns>
+        private static int GetAcquisitionNumberSuffix(string acquisitionFileNumber)
```

Comment:
Wouldn't it be better to make this an Extension Method of the PimsAcquisitionFile?

Reply:
yeah - can do that so the logic can be reused in the future

Reply:
Updated